dataland-keycloak/realms/master-realm.json

```diff
@@ -2393,7 +2393,7 @@
     "frontendUrl": "http://dataland-admin:6789/keycloak",
     "acr.loa.map": "{}"
   },
-  "keycloakVersion": "22.0.1",
+  "keycloakVersion": "22.0.3",
```

Comment:
The container keycloak version, the version in the other realm file and the keycloak-js version are still on 22.0.1. Is this intended?

Reply:
done

Reply:
Keycloak dockerfile line 14 is missing the update. This is the crucial point.